homework-exercises/src/main/java/io/codelex/flowcontrol/practice/LargestNumber.java

```diff
@@ -0,0 +1,33 @@
+package io.codelex.flowcontrol.practice;
+
+import java.util.Scanner;
+
+public class LargestNumber {
+
+    //TODO: Write a Java program to to find the largest of three numbers.
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+
+        System.out.print("Input the 1st number: ");
+        int num1 = in.nextInt();
+
+        System.out.print("Input the 2nd number: ");
+        int num2 = in.nextInt();
+
+        System.out.print("Input the 3rd number: ");
+        int num3 = in.nextInt();
+
+        if (num1 > num2) {
+            if (num1 > num3) {
```

Comment:
šo varētu pielikt ar `&&` arī klāt pie pirmā if
kā arī nebūs 2x jāraksta ` System.out.println("The greatest: " + num3);`